season-of-docs-proposal.md

```diff
@@ -0,0 +1,68 @@
+_This is a project proposal for Google's [Season of Docs](https://developers.google.com/season-of-docs) support._
+
+---
+
+# Improve Exercism's docs to encourage new contributions
+
+## About your organization
+
+Exercism is a not-for-profit organisation that helps people develop and improve their programming skills through practice and mentoring.
+Our platform is made up of over 200 GitHub repositories covering nearly 70 programming languages. 
+Everything is Open Source. Since launching in 2013 we have helped over 500,000 people become better programmers.
+
+Good communication is at the heart of Exercism. 
+Our platform is based around mentoring where strangers help each other understand and fill knowledge gaps.
+The product is also built by volunteers - so creating a fun, exciting and enticing place to build together is a key part of our mission, centered around empathy and kindness.
+
+## About your project
+
+### Your project’s problem
+
+One of our challenges is that Exercism being such a large project can make it intimidating and overwhelming to know where to start.
+Ensuring that contributors are able to find and adhere to our guidance, feel like they have a frictionless contribution experience, and that they can communicate clearly in the "Exercism way" with other contributors, is essential.
+
+As we have developed tighter quality-control and coherence between different parts of the project, there is more to know to get started, so the barrier to contribute has become slightly higher.
+We need to ensure that new contributors can find both clear overarching pathways to get started, and also the details when they need them.
+Good documentation is absolutely critical to ensuring that new contributors can find pathways to get started, and not feel frustration while trying to help.
+
+New contributors not only help Exercism grow, but more importantly improve the experience for hundreds of thousands of new students who use our platform each year.
+
+As part of the launch of our new version of the site launch soon, we have developed a whole "docs" section. You can preview it here: https://exercism.lol/docs
```

Comment:
Perhaps add a footnote that this is heavily WIP. The frontpage has a bunch of filler text and many of the links don't go anywhere.